src/components/VoiceRecorder/VoiceRecorder.constants.js

```diff
@@ -0,0 +1,2 @@
+export const ADD_RECORD = 'ADD_RECORD';
```

Comment:
We have a naming convention for actions: `cboard/VoiceRecorder/ADD_RECORD`, this way we can know in the debugger which component is responsible to which action

Reply:
To change ADD_RECORD on cboard/VoiceRecorder/ADD_RECORD? It doesn't work. I see the same kind of names in Notifications, 
export const HIDE_NOTIFICATION = 'HIDE_NOTIFICATION';
Can you say more about it? How it should be?